templates/base.html

```diff
@@ -9,6 +9,7 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="{% static 'dist/css/styles.css' %}">
```

Comment:
this is wrong. move the style files to `scripts/fe/src/assets` so it can be included in the compiled angular files